### PR TITLE
Bump ruby from 2.7.4 to 2.7.5 and bundler from 2.2.29 to 2.3.2

### PR DESCRIPTION
**Pull request by ybiquitous**
Via:

```console
$ rake bump:ruby'[2.7.5]'
```

Read the [release note](https://www.ruby-lang.org/en/news/2021/11/24/ruby-2-7-5-released/) for details.

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -1,7 +1,7 @@
 GEM
   remote: https://rubygems.org/
   specs:
-    aufgaben (0.8.5)
+    aufgaben (0.9.0)
```
Comment (ybiquitous) [inline Gemfile.lock:4]: https://github.com/ybiquitous/aufgaben/releases/tag/0.9.0

File: Gemfile.lock [hunk 2]
```diff
@@ -1,8 +1,8 @@
 GEM
   remote: https://rubygems.org/
   specs:
-    aufgaben (0.8.5)
-    cgi (0.2.0)
+    aufgaben (0.9.0)
+    cgi (0.3.1)
```
Comment (ybiquitous) [inline Gemfile.lock:5]: Ruby 2.7.5 includes CVE fixes about the CGI module.

**Comment by ybiquitous** [comment]
Oops! I'll fix the CI.

**Comment by ybiquitous** [comment]
The CI failure was due to an old Bundler version, so I've also bumped Bundler to the latest version.

I think this Bundler update doesn't include breaking changes. See also:

- https://github.com/rubygems/rubygems/blob/bundler-v2.3.2/bundler/CHANGELOG.md
- https://github.com/rubygems/rubygems/compare/bundler-v2.2.31...bundler-v2.3.2
- https://my.diffend.io/gems/bundler/2.2.31/2.3.2

File: base/Dockerfile [hunk 3]
```diff
@@ -38,7 +38,7 @@ RUN curl -fsSLO --compressed "${GIT_SOURCE_URL}" && \
 
 FROM debian:buster-20211011
 
-ENV GLOBAL_RUBY_VERSION 2.7.4
+ENV GLOBAL_RUBY_VERSION 2.7.5
```
Comment (radomirbosak) [inline base/Dockerfile:41]: > __[Goodcheck]__ Update CHANGELOG.md when you change a based Docker image. ([view](https://sider.review/gh/repos/200181683/pulls/741#issue-6417796))
>
> <details>
>
> | Rule |
> | ---- |
> | `sider.devon_rex.update_changelog_on_based_image_changed` |
>
> You can [close this issue](https://sider.review/gh/repos/200181683/pulls/741#issue-6417796) if no need to fix it. [Learn more](https://help.sider.review/getting-started/working-with-issues).
>
> </details>

changed

File: base/Dockerfile [hunk 4]
```diff
@@ -61,7 +61,7 @@ RUN test "$(git --version)" = "git version 2.33.1" && \
     rm -rf devon_rex
 
 # Install Ruby
-COPY --from=ruby:2.7.4-buster /usr/local /usr/local/
+COPY --from=ruby:2.7.5-buster /usr/local /usr/local/
```
Comment (radomirbosak) [inline base/Dockerfile:64]: > __[hadolint]__ COPY --from should reference a previously defined FROM alias ([view](https://sider.review/gh/repos/200181683/pulls/741#issue-6417788))
>
> <details>
>
> | Rule | Severity |
> | ---- | -------- |
> | `DL3022` | `warning` |
>
> References:
> - <https://github.com/hadolint/hadolint/wiki/DL3022>
>
> You can [close this issue](https://sider.review/gh/repos/200181683/pulls/741#issue-6417788) if no need to fix it. [Learn more](https://help.sider.review/getting-started/working-with-issues).
>
> </details>

We didn't change this part in this PR

File: dotnet/Dockerfile [hunk 5]
```diff
@@ -38,7 +38,7 @@ RUN curl -fsSLO --compressed "${GIT_SOURCE_URL}" && \
 
 FROM mcr.microsoft.com/dotnet/sdk:3.1.414-buster
 
-ENV GLOBAL_RUBY_VERSION 2.7.4
+ENV GLOBAL_RUBY_VERSION 2.7.5
```
Comment (radomirbosak) [inline dotnet/Dockerfile:41]: > __[Goodcheck]__ Update CHANGELOG.md when you change a based Docker image. ([view](https://sider.review/gh/repos/200181683/pulls/741#issue-6417797))
>
> <details>
>
> | Rule |
> | ---- |
> | `sider.devon_rex.update_changelog_on_based_image_changed` |
>
> You can [close this issue](https://sider.review/gh/repos/200181683/pulls/741#issue-6417797) if no need to fix it. [Learn more](https://help.sider.review/getting-started/working-with-issues).
>
> </details>

Updated.

**Review by radomirbosak — APPROVED** [review]
Looks good! :+1: Thank you!

**Pull request merged**